harvester/src/main/resources/application.properties

```diff
@@ -0,0 +1,2 @@
+# Port on which the harvester web application should be served
+server.port=9090
```

Comment:
please update the docs. in case there are none for harvester, create a README.md in harvester sub directory.